cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -313,32 +310,49 @@ def upload_splice_junctions_bed_to_scout(self, dry_run: bool, case_id: str) -> N
             dna_sample_id: str
             dna_cases: List[str]
             for dna_case_id in rna_dna_collection.dna_case_ids:
-                if self.status_db.get_case_by_internal_id(internal_id=dna_case_id).is_uploaded:
-                    LOG.info(
-                        f"Uploading splice junctions bed file for sample {dna_sample_name} "
-                        f"in case {dna_case_id} in Scout."
-                    )
+                LOG.info(
+                    f"Uploading splice junctions bed file for sample {dna_sample_name} "
+                    f"in case {dna_case_id} in Scout."
+                )
 
-                    if dry_run:
-                        continue
-
-                    self.scout_api.upload_splice_junctions_bed(
-                        file_path=splice_junctions_bed.full_path,
-                        case_id=dna_case_id,
-                        customer_sample_id=dna_sample_name,
-                    )
-                    LOG.info(
-                        f"Uploaded splice junctions bed file for sample {dna_sample_name} "
-                        f"in case {dna_case_id}."
-                    )
-                else:
-                    LOG.warning(
-                        f"Upload of splice junctions bed file for sample {dna_sample_name} in case {dna_case_id}"
-                        f" skipped - case has not finished uploading."
-                    )
+                if dry_run:
+                    continue
 
+                self.scout_api.upload_splice_junctions_bed(
+                    file_path=splice_junctions_bed.full_path,
+                    case_id=dna_case_id,
+                    customer_sample_id=dna_sample_name,
+                )
+                LOG.info(
+                    f"Uploaded splice junctions bed file for sample {dna_sample_name} "
+                    f"in case {dna_case_id}."
+                )
+        for upload_statement in self.rna_splice_junctions_upload_summary(rna_dna_collections):
+            LOG.info(upload_statement)
         LOG.info("Upload splice junctions bed file finished!")
 
+    def rna_splice_junctions_upload_summary(
+        self, rna_dna_collections: List[RNADNACollection]
+    ) -> List[str]:
+        log_statements: List[str] = []
```

Comment:
```suggestion
        upload_summary: List[str] = []
```
Consistency with function name